logic/src/androidMain/kotlin/com/wire/kalium/logic/CoreLogic.kt

```diff
@@ -51,6 +54,8 @@ actual class CoreLogic(
         kaliumConfigs.shouldEncryptData
     )
 
+    override val dataStore: DataStore<Preferences> get() = getDataStore(appContext)
```

Comment:
did you think about using `globalPreferences` instead of creating a separate `dataStore`?

Reply:
yes it's encrypted which makes it initialization slower

Reply:
well, you can make this one non-encrypted if you need to :)

Reply:
Yes correct but DataStore is preferred over SharedPreference.
https://android-developers.googleblog.com/2020/09/prefer-storing-data-with-jetpack.html

And they recently added support for KMM